src/test/java/io/github/externschool/planner/controller/MainControllerTest.java

```diff
@@ -1,35 +1,41 @@
 package io.github.externschool.planner.controller;
 
-
-import org.junit.Before;
+import org.hamcrest.Matchers;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
 import org.springframework.test.context.junit4.SpringRunner;
 import org.springframework.test.web.servlet.MockMvc;
-import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
 
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
-import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.view;
 
-
 @RunWith(SpringRunner.class)
+@WebMvcTest(controllers = MainController.class)
+@AutoConfigureMockMvc(secure=false)
 public class MainControllerTest {
-
+    @Autowired
     private MockMvc mockMvc;
 
-
-    @Before
-    public void setUp() {
-        mockMvc = MockMvcBuilders.standaloneSetup(new MainController()).build();
+    @Test
+    public void testLoginPageReturnsLoginFormView() throws Exception {
+        mockMvc.perform(MockMvcRequestBuilders.get("/login"))
+                .andExpect(status().isOk())
+                .andExpect(view().name("login"))
+                .andExpect(content().contentType("text/html;charset=UTF-8"))
+                .andExpect(content().string(Matchers.containsString("Login Form")));
     }
 
     @Test
-    public void testStartingPageReturnsGreetingView() throws Exception{
-        mockMvc.perform(get("/"))
+    public void testStartingPageReturnsGreetingView() throws Exception {
+        mockMvc.perform(MockMvcRequestBuilders.get("/"))
```

Comment:
For this case, we need to check the auth configuration. This page must be available without authorization